app/src/lib/git/commit.ts

```diff
@@ -4,6 +4,13 @@ import { Repository } from '../../models/repository'
 import { WorkingDirectoryFileChange } from '../../models/status'
 import { unstageAll } from './reset'
 
+/**
+ * Create a commit
```

Comment:
How do you feel about not having a comment for well-named functions and just keeping the parameters that need to be explained further?